### PR TITLE
Update shape of evaluation_reports

### PR DESCRIPTION
**Pull request by dethe**
Fix for #513 

You will need to run `npm run resetlocaldb`

### CODE REVIEW

File: api/controllers/case.js [hunk 1]
```diff
@@ -196,11 +196,11 @@ function getUpdatedCase(user, params, newCase, oldCase) {
   }
 
   // media lists
-  ["links", "videos", "audio", "evaluation_links"].map(
-    key => cond(key, as.media)
-  );
+  ["links", "videos", "audio"].map(key => cond(key, as.media));
   // photos and files are slightly different from other media as they have a source url too
-  ["photos", "files", "evaluation_reports"].map(key => cond(key, as.sourcedMedia));
+  ["photos", "files", "evaluation_reports", "evaluation_links"].map(key =>
```
Comment (ascott) [inline api/controllers/case.js]: `evaluation_links` should be like links, videos, and audio. it does not handle uploaded media.

File: migrations/migration_031.sql [hunk 2]
```diff
@@ -13,13 +13,21 @@ CREATE FUNCTION urls_to_links(urls text[]) RETURNS full_link[]
...
+
+ALTER TABLE cases ADD COLUMN evaluation_reports_full full_file[] DEFAULT '{}';
+ALTER TABLE cases ADD COLUMN evaluation_links_full full_file[] DEFAULT '{}';
 
-UPDATE cases SET evaluation_reports_full = urls_to_links(evaluation_reports) WHERE evaluation_reports <> '{}';
+UPDATE cases SET evaluation_reports_full = urls_to_files(evaluation_reports) WHERE evaluation_reports <> '{}';
 
 UPDATE cases SET evaluation_links_full =
-urls_to_links(evaluation_links) WHERE evaluation_links <> '{}';
+urls_to_files(evaluation_links) WHERE evaluation_links <> '{}';
```
Comment (ascott) [inline migrations/migration_031.sql]: evaluation_links does not do uploads and functions like links, videos and audio.

Comment (dethe) [reply migrations/migration_031.sql]: Yes, but it has the same fields as files, and photos. I modified the uploader code to not upload if the url is a link.

Comment (ascott) [reply migrations/migration_031.sql]: in the frontend code evaluation_links is treated the same as links, audio and videos and has the same fields as those.  Where as evaluation_reports are treated the same as files, photos. There are common fields between them, but i'm curious why we would treat evaluation_links the same as evaluation_reports on the backend...

File: api/controllers/case.js (hunk 1 above)
Comment (dethe) [reply api/controllers/case.js]: You're right, of course. I managed to mis-read the spec again.

**Pull request merged**